.github/workflows/pypi-wheel.yaml

```diff
@@ -0,0 +1,42 @@
+name: upload-wheel
+
+on:
+  push:
+    tags:
+      - "2.*"
+
+jobs:
+  maxos-and-windows:
+    runs-on: ${{ matrix.os }}-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ['3.5', '3.6', '3.7', '3.8']
+        os: [windows, macos]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+      - run: pip install wheel twine
+      - run: python setup.py build_ext
+        if: startsWith(runner.os, 'Windows')
+      - run: python setup.py bdist_wheel
+      - run: twine upload dist/*
+        env:
+          TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
+          TWINE_USERNAME: __token__
+
+  linux:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: RalfG/python-wheels-manylinux-build@v0.3.1
+        with:
+          python-versions: 'cp35-cp35m cp36-cp36m cp37-cp37m cp38-cp38'
+      - uses: actions/setup-python@v2
+      - run: pip install twine
+      - run: twine upload dist/*manylinux*
```

Comment:
Don't we need to build the wheels before uploading them with twine? I think a
```
      - run: python setup.py bdist_wheel
```
line got lost somewhere (or I don't understand how this is supposed to work).

Reply:
It's compiled by `uses: RalfG/python-wheels-manylinux-build@v0.3.1`

Reply:
I think having Linux wheels is not worth the trouble of maintaining the infrastructure running. By the fact that building them requires some non trivial steps I conclude that building Linux wheels is not as straightforward as on other platforms. Also, we have done without wheels on Linux for as long as beancount existed and I haven't heard too many complaints. Can you please remove this job?